src/router/index.js

```diff
@@ -71,7 +71,46 @@ const routes = [
 
 const router = createRouter({
   history: createWebHashHistory(),
+  base: process.env.FRONTEND,
   routes,
 });
 
+/*
+Router auth logic start
+
+Commented code below that works on `isAuthenticated` state and before every route:
+1. Redirects user to home if user is already logged in and visiting a page that is intended for guest (route.meta.guest)
+2. Redirects user to login if user is not authenticated and visits a page that requires authentication (route.meta.requiresAuth)
+
+Uncomment the lines below when the existing login checks that uses local storage are disabled.
+*/
+
+// router.beforeEach((to, from, next) => {
+//     if (to.matched.some((record) => record.meta.requiresAuth)) {
+//         if (store.getters.isAuthenticated) {
+//             next();
+//             return;
+//         }
+//         next("/login");
+//     } else {
+//         next();
+//     }
+// });
+
+// router.beforeEach((to, from, next) => {
+//     if (to.matched.some((record) => record.meta.guest)) {
+//         if (store.getters.isAuthenticated) {
+//             next("/posts");
+//             return;
+//         }
+//         next();
```

Comment:
didn't understand the logic behind this - what is the `/posts` page?

Reply:
This is a typo. Fixed in #142